dev/com.ibm.ws.opentracing_fat/publish/tckRunner/tck/blank-tck-suite.xml

```diff
@@ -0,0 +1,18 @@
+<!--Copyright (c) 2017 IBM Corporation and others. All rights reserved.
```

Comment:
You don't need one of these file. I just had one as an example as my main tck-suite had a
beanshell exclude in it.